File: Container With Most Water.js
var maxArea = function(height) {
  let i = 0;
let j = height.length-1;
let maximumArea = 0;
while(i<j){
 
  
  let area = Math.min(height[i],height[j])*(j-i);
   maximumArea = Math.max(area,maximumArea);
  if(height[i]<height[j])i++;
  else{
    j--;
  }
}
return maximumArea;
};

File: 11-container-with-most-water/container-with-most-water.py
class Solution(object):
    def maxArea(self, height):
        i = 0
        j = len(height) - 1
        max_area = 0
        while i < j:
            current_area = min(height[i] , height[j]) * (((j + 1) - (i + 1)))
            max_area = max(current_area , max_area)
            if height[i] < height[j]:
                i+=1
            else:
               j-=1
        return max_area

